proto/ibc/applications/interchain_accounts/v1/metadata.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package ibc.applications.interchain_accounts.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types";
+
+import "gogoproto/gogo.proto";
+
+// Metadata defines a set of protocol specific data encoded into the ICS27 channel version bytestring
+// See ICS004: https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#Versioning
+message Metadata {
+  string version                  = 1;
+  string controller_connection_id = 2 [(gogoproto.moretags) = "yaml:\"controller_connection_id\""];
+  string host_connection_id       = 3 [(gogoproto.moretags) = "yaml:\"host_connection_id\""];
+  string address                  = 4;
```

Comment:
Please add in-code documentation that this address field will be empty on controller address in INIT, and will be filled in by TRY by the host chain

Reply:
Done, good spot. Added inline doc comments to `Metadata` proto fields as well as inline comment in `InitInterchainAccount` entrypoint